src/test/java/com/solvd/micro9/users/TestUtils.java

```diff
@@ -0,0 +1,26 @@
+package com.solvd.micro9.users;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+import java.text.SimpleDateFormat;
+import java.util.Objects;
+
+public final class TestUtils {
```

Comment:
@eyakauleva you can declare your object mapper in `@TestConfiguration`

no need those utility classes

Reply:
@h1alexbel  I'm using TestUtils class in next feature for generating fake objects that's why I decided to put object mapper right here. Is this bad practice?

Reply:
@eyakauleva data generation is not the bad practice

but you can declare your object mapper bean inside your test configuration

WDYT?